graf2d/feynman/inc/TFeynman.h

```diff
@@ -0,0 +1,156 @@
+// @(#)root/feynman:$Id$
+// Author: Advait Dhingra    12/04/2021
+
+/*************************************************************************
+ * Copyright (C) 1995-2000, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#include <iostream>
+#include "TMath.h"
+#include "TVector3.h"
+#include <TLine.h>
+
+class TFeynman {
+
+	public:
+
+		// The constructor
+		TFeynman(Double_t canvasWidth, Double_t canvasHeight){
+			TCanvas *c1 = new TCanvas("c1", "c1", 10,10, canvasWidth, canvasHeight);
+   			c1->Range(0, 0, 140, 60);
+			gStyle->SetLineWidth(2);
+		}
+
+		// Fermions:
+		// Quarks:
+        // plots a Quark
+		void Quark(
+            Double_t x1, // x-Coordinate of starting point
```

Comment:
The data members should be documented the Doxygen way. see https://root.cern/for_developers/doxygen/#data-member-documentation